CMakeLists.txt

```diff
@@ -153,11 +153,12 @@ include_directories ( Framework/Kernel/inc )
 include_directories ( Framework/HistogramData/inc )
 include_directories ( Framework/Indexing/inc )
 include_directories ( Framework/Parallel/inc )
+include_directories ( Framework/Beamline/inc )
 include_directories ( Framework/Geometry/inc )
 include_directories ( Framework/API/inc )
 include_directories ( Framework/Types/inc )
 
-set ( CORE_MANTIDLIBS Kernel HistogramData Indexing Geometry API Types )
+set ( CORE_MANTIDLIBS Kernel HistogramData Indexing Beamline Geometry API Types )
```

Comment:
Should `Beamline` be here? `CORE_MANTIDLIBS` gets linked with everything passed this point. Can we just link it in the projects that need it? I'd like to get rid of this variable entirely at some point.

Reply:
@martyngigg  `Beamline` is fundamental and is used by `Geometry` therefore it needs to be wherever `Geometry` is used.

Reply:
@martyngigg `Beamline` is used by `Geometry` so technically speaking it is core. It needs to be wherever `Geometry` is used so I could take it out but I think it makes as much sense and removing `Geometry` from `CORE_MANTIDLIBS` at this stage.

Reply:
@LamarMoore Ok. Thanks for the explanation.